response.go

```diff
@@ -70,19 +81,19 @@ func (err connectionError) Timeout() bool {
 	return true
 }
 
-// serverError represents internal server errors.
+// ServerError represents internal server errors.
 // Implements `net.Error` interface.
-type serverError string
+type ServerError string
```

Comment:
What is purpose of exporting this type? I think no need to export this type, in case of using of an error of this type, better use type assertion to the `net.Error` interface.